### PR TITLE
Blocklist-component

### PR DESCRIPTION
**Pull request by ponochovny**
image

### CODE REVIEW

**Comment by aslife** [comment]
1. Do not use padding inside a component. Child component should have padding or marging.
2. If you open a white schema, you can see, when a block flipped, it gets a dark mode or huge shadow.

File: src/Components/BlockWithList/BlockWithList.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          theme={theme}
+          frontSide={(<div>frontside</div>)}
+          backSide={(<div>backSide</div>)}
+        />
+      </div>
+    </div>
+  )
+}
+
+BlockWithList.defaultProps = {
```
Comment (aslife) [inline src/Components/BlockWithList/BlockWithList.stories.tsx:62]: Why do you use default props for BlockWithList inside stories?
I think we should remove it from this stories

File: src/Components/BlockWithList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+export type Props = {
+  /** Define theme */
+  theme?: ETheme
+  frontSide: ReactNode
+  backSide: ReactNode
+}
+
+const defaultProps: Props = {
+  theme: ETheme.DARK,
+  frontSide: <div>frontside</div>,
```
Comment (aslife) [inline src/Components/BlockWithList/index.tsx:21]: We do not need create a default front side. It should be required.
Regarding backSide, if user add this props, we should use flip effect. If a user does not add backSide props. We should use only 1 side withou flip effect

File: src/Components/BlockWithList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+export type Props = {
+  /** Define theme */
+  theme?: ETheme
+  frontSide: ReactNode
+  backSide: ReactNode
+}
+
+const defaultProps: Props = {
+  theme: ETheme.DARK,
```
Comment (aslife) [inline src/Components/BlockWithList/index.tsx:20]: I thni we should add props for borderColor, because our mockup has diffrenet mode.

**Review by aslife — COMMENTED** [review]
We should take a look and discuss new comments

File: src/Components/BlockWithList/BlockWithList.scss [hunk 4]
```diff
@@ -0,0 +1,74 @@
+@import '../../Styles/constants.scss';
+@import '../../Styles/index.scss';
+
+.BlockWithList {
+    position: relative;
+    display: inline-block;
+    // width: 30%;
```
Comment (aslife) [inline src/Components/BlockWithList/BlockWithList.scss:7]: @ponochovny Remove unused code

File: src/Components/BlockWithList/BlockWithList.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,87 @@
+import React, { BaseSyntheticEvent, useState } from 'react'
+import { withKnobs, text } from '@storybook/addon-knobs'
+
+import BlockWithList from './index'
+import Button from '../OpiumButton'
+
+import { ETheme, sectionThemes } from '../../Constants/Types/theme.types'
+
+export const dropdownSelector = () => {
+  const items = [
```
Comment (aslife) [inline src/Components/BlockWithList/BlockWithList.stories.tsx:10]: Add props with data. We should get data from outside

Comment (ponochovny) [reply src/Components/BlockWithList/BlockWithList.stories.tsx:10]: So we have 2 props (frontSide and backSide). "const items = [..." is no needed.

**Review by aslife — APPROVED** [review]

**Pull request merged**